easybuild/easyconfigs/d/DIALOGUE/DIALOGUE-1.0-20230228-foss-2021b-R-4.2.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'DIALOGUE'
+local_commit = '7109f6a'
+# see DESCRIPTION to determine version,
```

Comment:
@boegel wonder whether it's worth clarifying here that you mean the file `DESCRIPTION` in the upstream code, not the `description` in the easyconfig, lest anyone make the dim mistake I did at first glance...